db/migrate/20190212110438_rename_image_timeline_entry_types.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class RenameImageTimelineEntryTypes < ActiveRecord::Migration[5.2]
+  def up
+    TimelineEntry.where(entry_type: "lead_image_updated")
+      .update!(entry_type: "lead_image_selected")
```

Comment:
should these be update_all?

Reply:
Good point - fixed :-).

Reply:
Thanks - not sure if you caught my other comment about db/schema.rb

Reply:
Oops - it was also modifying another line and I checked-it-out by mistake.